pkg/restapi/did/operation/operations.go

```diff
@@ -145,21 +177,18 @@ func addRequestHeaders(req, newReq *http.Request) {
 	}
 }
 
-func (o *Operation) writeErrorResponse(rw http.ResponseWriter, status int, msg string) {
-	log.Warnf("proxy returning status code: %d, error: %s", status, msg)
+func writeBadRequestResponse(rw http.ResponseWriter, msg string) {
```

Comment:
No reason to change this - you may want to use it for status other than bad request